src/test/scala/io/github/pauljamescleary/petstore/infrastructure/repository/doobie/package.scala

```diff
@@ -16,12 +17,28 @@ package object doobie {
       cfg.password           // password
     )
 
+  /*
+   * Create an h2 DataSource which flyway can use to initialize the database.
```

Comment:
We're considering whether it makes sense to hardcode H2 in test here.

Reply:
And apparently it does not because flyway does actually autoselect the correct driver given just the URL somehow.

Reply:
Ah, there it is: https://github.com/flyway/flyway/blob/0260b5c37eb0c8f3bbc331c1e293710e32324deb/flyway-core/src/main/java/org/flywaydb/core/internal/jdbc/DriverDataSource.java#L280